common/app/views/support/Profile.scala

```diff
@@ -129,6 +129,7 @@ object FacebookOpenGraphImage extends ShareImage(FacebookShareImageLogoOverlay.i
 
 object EmailImage extends Profile(width = Some(580), autoFormat = false) {
   override val qualityparam = "q=40"
+  val knownWidth = 580
```

Comment:
Yeah I agree it's a bit rubbish duplicating that value. Really what I want is to be able to say that `width` is not optional on this subtype - because it's an `Option` I'd need to `.getOrElse` and provide a fallback value wherever I use it, which would then duplicate the `width` value anyway. So I thought duplicating the value in one place and right next to the original was preferable. However they may be a much nicer way to do this - open to suggestions